### DISCUSSION TITLE
Possible to use widgets on Statamic dashboard?

### DISCUSSION THREAD

**Discussion by Web10-Joris** [duncanmcclean/simple-commerce#907] [Feature Requests]
I was wondering if it is possible to use (for example) the order-overview widget on the statamic home dashboard together with other created widgets?

**Comment by duncanmcclean** [comment]
Hi @Web10-Joris,

Not currently - however, I'd be happy to merge a PR that makes that possible!

**Comment by duncanmcclean** [comment]
This will be possible in Simple Commerce v6 (#963), which will be released in the next month or two.
